dts/bindings/opamp/opamp-controller.yaml

```diff
@@ -17,6 +18,93 @@ properties:
     description: |
       Selects opamp functional mode.
 
+      a) inverting mode
```

Comment:
Maybe order in the same order used in the enumerated list above.

Reply:
Thnx. Done